scripts/machine_translation/evaluate_transformer.py

```diff
@@ -29,14 +30,29 @@ def parse_args():
                         help='The source corpus for evaluation.')
     parser.add_argument('--tgt_corpus', type=str, default=None,
                         help='The target corpus for evaluation.')
+    parser.add_argument('--src_normalizer', choices=['no', 'moses'],
+                        default='moses', help='The sentence normalizer that will be '
+                                              'used to normalize the source sentence.')
```

Comment:
I'm not sure whether "normalize" is a helpful concept, as it doesn't tell me what it does

Reply:
if it's already a convention elsewhere I'm ok to learn that (so that it's easier to communicate with that group). Just want to point out the semantics of "normalize" isn't as clear as "tokenize"

Reply:
Usually, it means some pre-filtering steps before tokenization. For example, the definition of lossless tokenization is 

```
Decode(Encode(normalize(text))) == normalize(text)
```

Ref: Sentencepiece paper: https://arxiv.org/pdf/1808.06226.pdf, Page2.